js/models/conversations.js

```diff
@@ -548,6 +550,14 @@
                 };
             }.bind(this));
 
+            // Some messages we're marking read are local notifications with no sender
+            read = _.filter(read, function(m) {
+                return Boolean(m.sender);
```

Comment:
In this case, we're not actually dealing with a model, but a little object with just two keys. In all other cases I did change it over to `m.isIncoming()`